app/views/guests/show.html.erb

```diff
@@ -0,0 +1,16 @@
+<h1>Hi, <%=@guest.name%>. Here is your past stay information.</h1>
+<% @rooms.each do |room|%>
+  <h3><%=room.suite%> at the <%=room.hotel_name%></h3>
+  <p>Room Type: <%=room.suite%><p>
+  <p>Nightly Rate: $<%=room.rate%><p>
+  <p>Hotel: <%=room.hotel_name%><p>
+<%end%>
+<br>
+<div id="add_stay">
+  <h2>Add a Stay:</h2>
+  <%= form_tag("/guests/#{@guest.id}", method: :get, local: true) do%>
```

Comment:
FYI - `form_tag` is a deprecated (old) view helper method; we would rather you use `form_with`. We will also be adding to the features of the `form_with` view helper in class soon, so definitely get used to that method instead.

Reply:
If I were to look at this route separate from the functionality implemented, I would assume this form is responsible for retrieving an existing Guest instance from the database when I hit "Submit". But, that's not what the user story is asking for, and it's not what your form is doing either - it's submitting a Room ID to be related to this existing Guest. 

I think a better route overall would be
```
post "/guest/:id/rooms", to: "guest_rooms#create"
# alternatively: 
post "/guest_rooms", to: "guest_rooms"#create"
```
If you did it the 2nd way I listed above, you'd have to use a hidden field for the `guest_id` in your form, but either would be correct.